feature/experimental/tunnel/otg_tests/tunnel_interface_based_disable_enable_test/tunnel_interface_disable_enable_test.go

```diff
@@ -0,0 +1,1558 @@
+//  *   gre/source/interface/
+//
+//  ## Telemetry Parameter coverage
+//
+//  *   state/counters/in-pkts
+//  *   state/counters/in-octets
+//  *   state/counters/out-pkts
+//  *   state/counters/out-octets
+//  *   state/counters/in-error-pkts
+//  *   state/counters/in-forwarded-pkts
+//  *   state/counters/in-forwarded-octets
+//  *   state/counters/in-discarded-pkts
+//  *   state/counters/out-error-pkts
+//  *   state/counters/out-forwarded-pkts
+//  *   state/counters/out-forwarded-octets
+//  *   state/counters/out-discarded-pkts
+//
+//  ## Topology:
+//  *   otg:port1 <--> port1:dut1:port3 <--> port3:dut2:port5<--->otg:port5
+//  *   otg:port2 <--> port2:dut1:port4 <--> port4:dut2:port6<--->otg:port6
+
+type parameters struct {
+	rtIntf1Ipv4Add  string
+	rtIntf2Ipv4Add  string
+	rtIntf5Ipv4Add  string
+	rtIntf6Ipv4Add  string
+	rtIntf1MacAdd   string
+	rtIntf2MacAdd   string
+	rtIntf5MacAdd   string
+	rtIntf6MacAdd   string
+	r0Intf1Ipv4Add  string
+	r0Intf2Ipv4Add  string
+	r0Intf3Ipv4Add  string
+	r0Intf4Ipv4Add  string
+	r0Fti0Ipv4Add   string
+	r0Fti1Ipv4Add   string
+	r0Fti2Ipv4Add   string
+	r0Fti3Ipv4Add   string
+	r0Fti4Ipv4Add   string
+	r0Fti5Ipv4Add   string
+	r0Fti6Ipv4Add   string
+	r0Fti7Ipv4Add   string
+	r0Lo0Ut0Ipv4Add string
+	r0Lo0Ut1Ipv4Add string
+	r0Lo0Ut2Ipv4Add string
+	r0Lo0Ut3Ipv4Add string
+	ipv4Mask        uint8
+	ipv4FullMask    uint8
+	r1Intf5Ipv4Add  string
+	r1Intf6Ipv4Add  string
+	r1Intf3Ipv4Add  string
+	r1Intf4Ipv4Add  string
+	r1Fti0Ipv4Add   string
+	r1Fti1Ipv4Add   string
+	r1Fti2Ipv4Add   string
+	r1Fti3Ipv4Add   string
+	r1Fti4Ipv4Add   string
+	r1Fti5Ipv4Add   string
+	r1Fti6Ipv4Add   string
+	r1Fti7Ipv4Add   string
+	r1Lo0Ut0Ipv4Add string
+	r1Lo0Ut1Ipv4Add string
+	r1Lo0Ut2Ipv4Add string
+	r1Lo0Ut3Ipv4Add string
+	rtIntf1Ipv6Add  string
+	rtIntf2Ipv6Add  string
+	rtIntf5Ipv6Add  string
+	rtIntf6Ipv6Add  string
+	r0Intf1Ipv6Add  string
+	r0Intf2Ipv6Add  string
+	r0Intf3Ipv6Add  string
+	r0Intf4Ipv6Add  string
+	r0Fti0Ipv6Add   string
+	r0Fti1Ipv6Add   string
+	r0Fti2Ipv6Add   string
+	r0Fti3Ipv6Add   string
+	r0Fti4Ipv6Add   string
+	r0Fti5Ipv6Add   string
+	r0Fti6Ipv6Add   string
+	r0Fti7Ipv6Add   string
+	r0Lo0Ut0Ipv6Add string
+	r0Lo0Ut1Ipv6Add string
+	r0Lo0Ut2Ipv6Add string
+	r0Lo0Ut3Ipv6Add string
+	ipv6Mask        uint8
+	ipv6FullMask    uint8
+	r1Intf5Ipv6Add  string
+	r1Intf6Ipv6Add  string
+	r1Intf3Ipv6Add  string
+	r1Intf4Ipv6Add  string
+	r1Fti0Ipv6Add   string
+	r1Fti1Ipv6Add   string
+	r1Fti2Ipv6Add   string
+	r1Fti3Ipv6Add   string
+	r1Fti4Ipv6Add   string
+	r1Fti5Ipv6Add   string
+	r1Fti6Ipv6Add   string
+	r1Fti7Ipv6Add   string
+	r1Lo0Ut0Ipv6Add string
+	r1Lo0Ut1Ipv6Add string
+	r1Lo0Ut2Ipv6Add string
+	r1Lo0Ut3Ipv6Add string
+	flow1           string
+	flow2           string
+	flow3           string
+	flow4           string
+	trafficDuration int64
+	trafficRate     int64
+}
+
+func GetNetworkAddress(t *testing.T, address string, mask int) string {
```

Comment:
Go style guide says functions shouldn't start with "Get". "Fetch" or "Lookup" here is fine